distributions/otelcol-contrib/.goreleaser-build.yaml

```diff
@@ -37,6 +24,40 @@ builds:
       - -trimpath
     env:
       - CGO_ENABLED=0
+  - id: otelcol-contrib-darwin
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    dir: _build
+    binary: otelcol-contrib
+    ldflags:
+      - -s
+      - -w
```

Comment:
since the ldflags are used multiple times now, they could be put into an env var and re-used